postgres-persistence/src/test/java/com/netflix/conductor/postgres/performance/PerformanceTest.java

```diff
@@ -417,10 +417,13 @@
 //    }
```

Comment:
Can the commented code be removed?

Reply:
This test cannot be automated and is only available for manual testing. Further, this test needs to be migrated to SB which we leave upto the users of this module to take on since it is not essential to running/testing Conductor with postgres.